models/message.go

```diff
@@ -61,6 +61,9 @@ type Attachment struct {
 
 	AudioURL string `json:"audio_url,omitempty"`
 	VideoURL string `json:"video_url,omitempty"`
+	
+	Actions []AttachmentAction `json:"actions,omitempty"`
```

Comment:
Does `omitempty` work on slices?

Reply:
it should.  It should just omit the entire slice if its empty.